src/utils/utils.tsx

```diff
@@ -25,20 +27,36 @@ export const getGeography = (city: string[] | null | undefined, state: string[]
     return Translate("Not Reported");
   }
 
-  function renderOutGeography(geo: string[] | null | undefined) {
+  function renderOutGeography(geo: string[] | string | null | undefined) {

```

Comment:
This function is used to format the location field in the study modal right? That's not super apparent based on the name imo, could we either add a comment or change the function name